docs/manage/_reports.rst

```diff
@@ -50,21 +54,26 @@ Open the **Quizzes** sub-tab from the main **Reports** view to access the progre
       .. figure:: img/exam-report-detail.png
         :alt: 
 
-
 #. Click on the attempt icons to see the answers for each one. In the screenshot above learner **Abby L.** had 3 attempts on *Question 8*: the first one was incorrect, the second one (selected) was correct, but the third and finally submitted one was incorrect.
 
-.. _exercise_progress:
+    .. _exercise_progress:
+
+    .. note:: 
+      
+      **Review Exercise Progress**
+
+      When a learner answers exercise questions in Kolibri, the progress bar below the exercise is taking into account only the most recent given answers, meaning that the learner must complete the required number of correct answers (*check marks*) **in the row** for the exercise to be considered completed.
+
+        .. figure:: img/get-5-correct.png
+            :alt: If the exercise requires 5 correct check marks, learner must provide 5 correct answers one after another. Marks for each answer are located behind the button Check
 
-.. note:: 
-	
-	**Review Exercise Progress**
+            The correct answers (green check marks) in the image above are not in a row; this exercise will be completed only after the learner gives 5 correct answers one after another.
 
-	When a learner answers exercise questions in Kolibri, the progress bar below the exercise is taking into account only the most recent given answers, meaning that the learner must complete the required number of correct answers (*check marks*) **in the row** for the exercise to be considered completed.
+#. Click :guilabel:`OPTIONS` button and select **Preview** or **Edit details** option if you want to view or edit the quiz details directly from its report. Editing the quiz details from here will have the same results as from the **Plan** tab (|edit| icon).
 
-		.. figure:: img/get-5-correct.png
-		    :alt: If the exercise requires 5 correct check marks, learner must provide 5 correct answers one after another. Marks for each answer are located behind the button Check
+    .. figure:: img/edit-quiz-from-report.gif
+      :alt: 
```

Comment:
Is there an `:alt:` text missing here? My diff seems to say so.

Reply:
The `get-5-correct.png` image and its `:alt:` are moved to lines 67 & 68.

`:alt:` for `edit-quiz-from-report.gif` is missing intentionally, as in several other places in the docs. Describing this image (a GIF animation) would entail re-writing the text which is already present above, and be a repetition for a screen-reader users.

Reply:
I was wondering if it was a mistake, thanks for explaining! Not trying to be a screen reader expert here -- that's you :1st_place_medal: